### ISSUE TITLE
iOS push notification issue?

### ISSUE THREAD

**Issue by EddyVerbruggen** [tahir-jamil/seefarma#1]
What's the problem exactly, and what do I do to reproduce it?

**Comment by tahir-jamil** [comment]
well repo is working completely no issue with the repo

**Comment by tahir-jamil** [comment]
notifications are also received in onMessageReceivedCallback

**Comment by tahir-jamil** [comment]
for android i use local notification and its working fine for android but for ios local notifications are not show either by use of local notification plugin not by using         firebase.registerForInteractivePush(this.model);

**Comment by tahir-jamil** [comment]
i feel i am not using registerForInteractivePush function in right way so just need some guide about it

**Comment by tahir-jamil** [comment]
DO you find some thing eddy ?

**Comment by EddyVerbruggen** [comment]
Not yet. You must realize this is a free service I'm offering so I can't just drop everything.

**Comment by EddyVerbruggen** [comment]
I fail to see what the issue is. You mention 'registerForInteractivePush' but I don't see it being used in this repo at all.

Note that I do receive notifications on iOS (Android not tested) both in the foreground and background, but I had to rename `fiver17.entitlements` to `seefarma.entitlements` (as well as the reference in `build.xcconfig`.

**Comment by tahir-jamil** [comment]
are you receiving like local notification in ios ?

**Comment by tahir-jamil** [comment]
i want to show local notification for both android and ios for android its working but in my case ios is not working

**Comment by tahir-jamil** [comment]
i will test it again

**Comment by EddyVerbruggen** [comment]
So the issue is with local notifications on iOS? I did not test that. How can I trigger one in your app?

**Comment by tahir-jamil** [comment]
well i am using your plugin of local notification and it works for Android but i have see some post in which i came to know that i cannot use local notification with firebase and firebase provide us their own way to show local notifications

**Comment by tahir-jamil** [comment]
and you have post a docs for showing local notification

**Comment by tahir-jamil** [comment]
i have tried this but i think i am not implementing it in a right way all of this is in home.component.ts

**Comment by tahir-jamil** [comment]
https://github.com/EddyVerbruggen/nativescript-plugin-firebase/blob/master/docs/MESSAGING.md#ios-background-notification-example

**Comment by EddyVerbruggen** [comment]
I don’t understand the problem. What exactly is not working on iOS?

**Comment by tahir-jamil** [comment]
when i receive notification from firebase i want to show a local notification in ios

**Comment by tahir-jamil** [comment]
this is only the issue in ios only

**Comment by EddyVerbruggen** [comment]
Yes but what do you mean by ‘a local notification’. Does it need to be interactive (buttons, text entry), or do you want it to show when the app is in the foreground, etc?

Do you actually need a local notification, or does the remote notification need to be shown while it isn’t?

**Comment by tahir-jamil** [comment]
no i need local notification when the app is in foreground

**Comment by tahir-jamil** [comment]
no i need local notification when the app is in foreground

**Comment by tahir-jamil** [comment]
and i only need to notify the user that a new notification is received nothing else no interaction

**Comment by EddyVerbruggen** [comment]
So let me summarize the issue for you: everything works fine, except push notifications are not shown when the iOS app is in the foreground. That's it, right?

Then remove the local-notifications plugin and set `showNotificationsWhenInForeground` to `true` as shown [here](https://github.com/EddyVerbruggen/nativescript-plugin-firebase/blob/master/docs/MESSAGING.md#always-show-notifications-when-the-application-is-in-foreground).

If you allow me commit access to this repo I'll push the required changes myself because I have your app working as described.

**Comment by tahir-jamil** [comment]
ya wait let me allow

**Comment by tahir-jamil** [comment]
added 
Also if i remove local notification plugin than how can i use local notification in android ?

If their is no any solution for using both android and ios local notifications in a same repo i can handle them separately one for android and one for ios

**Comment by EddyVerbruggen** [comment]
You said it was a problem on iOS so I checked iOS. I would appreciate if you stop spamming me now.

**Comment by tahir-jamil** [comment]
ok sorry

**Comment by tahir-jamil** [comment]
thanks for this
